storybookjs/storybook
# Controls: Remove auto inference and add to CLI template

Issue: #14014

## What I did

Seems like the controls matchers as a preset were causing confusion, so this PR removes them and explicitly adds them in the CLI templates for discoverability.

## How to test

1 - checkout the branch
2 - run yarn bootstrap && yarn start
3 - Check the examples at `examples/official-storybook/stories/addon-controls.stories.tsx`, they should still work
